### PR TITLE
Add Preview IP filtering, update screenshots on page, add visual walkthrough

### PR DESCRIPTION
**Pull request by dacharyc**
Unfortunately I had a problem with this on my local when I added the "visual walkthrough" style at the bottom, even though I basically just copied it from the "Setting up Tugboat -> Add Repos to the Project" page where it works fine. Not sure what the issue is. I thought maybe it was something on my local but it looks like it's still borked on the Tugboat Preview.

Want me to make a separate commit with just the Preview IP filtering content, @bchavet , so we can merge that, at least, and then deal with the "Visual Walkthrough" another time? Or do you have ideas for an easy fix to the page so we can merge it in its entirety? (And/or feedback for the Preview IP filtering part?)

### CODE REVIEW

File: content/setting-up-tugboat/select-repo-settings.md [hunk 1]
```diff
@@ -125,6 +126,12 @@ credentials from within the repo settings.
 
 ![Authenticate with a Docker registry](../../_images/authenticate-with-a-docker-registry-add-credentials.png)
 
+### Preview IP Filters
```
Comment (bchavet) [inline content/setting-up-tugboat/select-repo-settings.md:129]: This needs to have more information, which I can get to you when I have a chance

* How to format the config values
* How the filtering works
  * Default behavior
  * Behavior when one or more filters are present

Comment (bchavet) [inline content/setting-up-tugboat/select-repo-settings.md:129]: This is about to be an Enterprise-only feature. Do we need to call that out somehow? We don't really have a precedent set for documenting features available at different tiers.

Comment (dacharyc) [reply content/setting-up-tugboat/select-repo-settings.md:129]: Yes. For now, I'm thinking about using a shortcode to get one of the colored boxes, and call it out as Enterprise-only, ala: https://learn.netlify.app/en/shortcodes/notice/

Maybe a new one that we only use to denote Enterprise features?

Or if we think we may ever have other tiers and want to denote this is a "tier-specific" feature and not just default to "Enterprise," there may be benefit in using more general language in the header part. I'll ponder some options and maybe we can do an after-talk on the scrum on Wednesday.

Comment (dacharyc) [reply content/setting-up-tugboat/select-repo-settings.md]: Regarding this, #213 creates a custom "notice" style so we can add a callout to denote this as an enterprise-only feature.

I'll also add something to the marketing site for this once we get the docs all set up.

Comment (bchavet) [reply content/setting-up-tugboat/select-repo-settings.md]: I like the call-out in #213

Comment (bchavet) [reply content/setting-up-tugboat/select-repo-settings.md]: > How to format the config values

The config can accept any of the following

* Any valid IPv4 subnet, using CIDR notation
* Any valid IPv6 subnet, using CIDR notation
* Any valid IPv4 address. We will automatically append a /32 CIDR subnet mask
* Any valid IPv6 address. We will automatically append a /128 CIDR subnet mask

Information about CIDR notation: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing

> How the filtering works

When no filters are provided, no filtering is done. Preview URLs are publicly accessible to anyone that has the link (this is how Tugboat has always worked)

If one or more filters are provided, ONLY requests coming from an address that matches one of those filters are allowed access to the Preview URLs. Any other requests will be given a 404 - Preview Not Found.

An address "matches" if it is a member of any of the subnets in the list of filters.

> Bonus, not sure if you want to write this up or not

We also support DENY filters, but those can only be managed by the API or CLI. _deny_ filters are processed AFTER _allow_ filters.

If both _allow_ and _deny_ filters are present, a source IP must match an _allow_ filter, but must NOT match a _deny_ filter in order to access a Preview URL. 

If only _deny_ filters are present, a source IP must NOT match a _deny_ filter in order to access a Preview URL.

_deny_ filters are not very useful on their own, but provide a way for advanced users to allow access to a large subnet, and then specify exceptions to that allow list.

**Comment by bchavet** [comment]
test this please

**Pull request merged**